src/rustdoc-json-types/lib.rs

```diff
@@ -733,17 +1170,25 @@ pub struct OpaqueTy {
     pub generics: Generics,
 }
 
+/// A global variable declaration.
 #[derive(Clone, Debug, PartialEq, Eq, Hash, Serialize, Deserialize)]
 pub struct Static {
+    /// The type of the static.
     #[serde(rename = "type")]
     pub type_: Type,
+    /// This is `true` for mutable statics, declared as `static mut X: T = f();`
     pub mutable: bool,
+    /// The stringified expression for the initial value. It's not guaranteed that
+    /// it'll match the actual source code for the initial value.
     pub expr: String,
 }
 
+/// A primitive type declaration. Declarations of this kind can only come from the core library.
 #[derive(Clone, Debug, PartialEq, Eq, Hash, Serialize, Deserialize)]
 pub struct Primitive {
+    /// The name of the library.
```

Comment:
??, I think this should be the name of the primitive.

But also, this seems redundant with having `Item::name`, and should probably be removed (in a different PR).